sass/components/_buttons.scss

```diff
@@ -119,7 +119,7 @@ button {
     &:active,
     &--active,
     &--active:hover {
-        background: rgba(var(--button-bg-rgb), 0.08);
+        background: rgba(var(--center-channel-color-rgb), 0.08);
```

Comment:
`--button-bg-rgb` is actually the appropriate colour here per current design specs.

![image](https://user-images.githubusercontent.com/3245614/105416232-d7347d80-5c07-11eb-9f87-695a153f60a4.png)

So I suggest we revert this change:

```suggestion
        background: rgba(var(--button-bg-rgb), 0.08);
```